### PR TITLE
[StructuralMechanicsApplication] Adding patch tests for prism and tetrahedra

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by loumalouomega** [comment]
Anyone, this is just new tests, it is a good thing :)

File: applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py [hunk 1]
```diff
@@ -251,7 +259,7 @@ def test_SmallDisplacementElement_2D_triangle(self):
         #checking consistent mass matrix
         M = KratosMultiphysics.Matrix(0,0)
         mp.Elements[1].CalculateMassMatrix(M,mp.ProcessInfo)
-        Area = mp.Elements[1].GetArea()
+        Area = mp.Elements[1].GetGeometry().Area()
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py:262]: :+1:

File: applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ def _apply_material_properties(self,mp,dim):
         mp.GetProperties()[1].SetValue(KratosMultiphysics.POISSON_RATIO,0.3)
         mp.GetProperties()[1].SetValue(KratosMultiphysics.THICKNESS,1.0)
         mp.GetProperties()[1].SetValue(KratosMultiphysics.DENSITY,1.0)
+        #mp.GetProperties()[1].SetValue(StructuralMechanicsApplication.INTEGRATION_ORDER,5)
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py:41]: ?

**Review by philbucher — APPROVED** [review]
sorry missed it somehow

File: applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py (hunk 2 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py:41]: I was testing if the integration order affected

**Pull request merged**